src/EventStore.Common/Log/SeriLogger.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using Serilog.Events;
+
+namespace EventStore.Common.Log
+{
+    public class SeriLogger : ILogger
+    {
+        private readonly Serilog.ILogger _logger;
+        private string _name;
+
+        public SeriLogger(string name)
+        {
+           _logger = Serilog.Log.ForContext(name, null);
```

Comment:
should be `Serilog.Core.Constants.CnotextName or similar, name` - see impl of ForContext

Reply:
Thanks @bartelink for pointing out. I will make another commit in this PR.

Reply:
SourceContext gives the full namespace but to keep backwards compatibility with the previous logging we are using the same class name variable used before.